### ISSUE TITLE
ihpool.com is down and miner couldn't connect to server

### ISSUE THREAD

**Issue by leicao-me** [hpool-dev/chia-miner#679]
I'm in the US. 
1) ihpool.com can be visited (This site can’t be reached. The connection was reset.)
2) the hpool miner couldn't reach the server.

This happens in the morning after 4AM PDT. I tried to add "line: jp" to config.yaml to switch to Japanese line but it doesn't work.

**Comment by leicao-me** [comment]
My friends in China don't have this issue. Even my local friend with another internet service provider doesn't have this issue. 
Either my home internet or phone data internet can't connect to hpool now.

**Comment by leicao-me** [comment]
Today, everything magically went back to be working. It's either the internet service provider's or hpool regional server offline's issue.

**Comment by Michael881188** [comment]
I'm facing the same issue since yesterday(Poland),it was working for over 2 months without any issue, nothing in my setup or service provider's changed.I cannot access web page and my miner is just throwing errors:
'time="2021-07-21T23:24:26+02:00" level=error msg="connect fail" err="read tcp 192.168.0.170:49660->203.107.53.48:443: wsarecv: The existing connection was abruptly closed by the remote host." f7="{loggers.go,logging.CPrint,156}" f8="{server.go,websocket.(*Server).connect,89}" f9="{server.go,websocket.(*Server).check,65}" tid=105 url="wss://chia-sh4.hpool.com/x-proxynode/ws"
'
I tried:
- to use VPN
- switching DNS server to cloudflare(1.1.1.1)
- switching to japan

**Comment by leicao-me** [comment]
> I'm facing the same issue since yesterday(Poland),it was working for over 2 months without any issue, nothing in my setup or service provider's changed.I cannot access web page and my miner is just throwing errors:
> 
> 'time="2021-07-21T23:24:26+02:00" level=error msg="connect fail" err="read tcp 192.168.0.170:49660->203.107.53.48:443: wsarecv: The existing connection was abruptly closed by the remote host." f7="{loggers.go,logging.CPrint,156}" f8="{server.go,websocket.(*Server).connect,89}" f9="{server.go,websocket.(*Server).check,65}" tid=105 url="wss://chia-sh4.hpool.com/x-proxynode/ws"
> 
> '
> 
> I tried:
> 
> - to use VPN
> 
> - switching DNS server to cloudflare(1.1.1.1)
> 
> - switching to japan
> 
> 

Yeah, exactly the same issue as mine. I guess hpool is maintaining their servers or global server accessibility. Mine started working today. I reversed everything I tried, and after a night it started working by itself.

**Comment by Michael881188** [comment]
It's working today. No idea what happened.
